diagrams/app_diagrams_generator.rb

```diff
@@ -0,0 +1,219 @@
+require 'yaml'
+require 'fileutils'
+
+class AppDiagramGenerator
+  def initialize
+    @repository_path = Dir.getwd
```

Comment:
This doesn't need to be an instance variable. I think only calling `YAML.load_file("data/applications.yml")` would also work, we don't need to remember the path I think.